test/tbb/test_task_arena.cpp

```diff
@@ -1749,6 +1749,7 @@ struct enqueue_test_helper {
     std::atomic<std::size_t>& my_task_counter;
 };
 
+#if 0
```

Comment:
Do we need test below?

Reply:
We need them, it is some experiments, I will revert